docs/References.bib

```diff
@@ -158,6 +158,12 @@ @article{Boyle2019kee
       SLACcitation   = "%%CITATION = ARXIV:1904.04831;%%"
 }
 
+@book{Chandrasekhar1939,
+  author = "Chandrasekhar, Subrahmanyan",
+  title  = "An introduction to the study of stellar structure",
+  year   = "1939"
```

Comment:
Another, maybe easier to get a hold of references is Teukolsky and Shapiro

Reply:
But that one does not explicitly give the Gamma=2 (alternatively, n=1) solution that is implemented. I'll use both?

Reply:
Oh, sorry, yes I meant referencing both :)